### PR TITLE
Fix premerge-checks to lint against develop

### PR DESCRIPTION
**Pull request by manupak**
This commit fixes the base commit being the commit just before 1 to develop branch. In that way, the whole PR is linted.

### CODE REVIEW

File: mlir/utils/jenkins/static-checks/premerge-checks.py [hunk 1]
```diff
@@ -17,21 +17,21 @@
...
 
 
 def get_diff(base_commit) -> Tuple[bool, str]:
-  r = subprocess.run(f'/opt/rocm/llvm/bin/git-clang-format {base_commit}', shell=True)
-  if r.returncode != 0:
-    r = subprocess.run(f'git checkout -- .', stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+  diff_run = subprocess.run(f'/opt/rocm/llvm/bin/git-clang-format --diff {base_commit}', shell=True)
+  if diff_run.returncode != 0:
     return False, ''
-  diff_run = subprocess.run(f'git diff -U0 --no-prefix --exit-code', stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
```
Comment (krzysz00) [inline mlir/utils/jenkins/static-checks/premerge-checks.py:32]: I think the point of this diff call is to show the changes that come up when you run clang-format. We might want to keep it, but it's not urgent given our workflow.

**Review by krzysz00 — APPROVED** [review]
Approved

File: mlir/utils/jenkins/static-checks/premerge-checks.py (hunk 1 above)
Comment (manupak) [reply mlir/utils/jenkins/static-checks/premerge-checks.py:32]: Thats spelled git clang-format --diff and its still maintainined in the same way and one does not need to do the changes and revert

Comment (manupak) [reply mlir/utils/jenkins/static-checks/premerge-checks.py:32]: (See my new diff_run var)

Comment (krzysz00) [reply mlir/utils/jenkins/static-checks/premerge-checks.py:32]: Ah, cool. in which case, we're good

**Pull request merged**